compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -3297,7 +3297,7 @@
                 },
                 {
                     "name": "receiveWorkers",
-                    "type": "SimpleNameReferenceNode"
+                    "type": "Node"
```

Comment:
Don't we have parser test cases for multiple receive action?

Reply:
We do have. https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-parser/src/test/resources/actions/receive-action/receive_action_source_01.bal